### DISCUSSION TITLE
Loading plugins without a turtle?

### DISCUSSION THREAD

**Discussion by ghost** [monocasual/giada#688] [Q&A] [answered]
My plugins folder is just a bunch of VST2 DLLs. When I scan it, nothing happens, and the logs complain that the plugins have no manifest:

> lilv_world_load_file(): error: Error loading file `file:///D:/mus/Music/Plugins/SoulForce-vst.dll/manifest.ttl'

Other hosts don't seem to complain about a missing manifest, so what gives? Is there a way to tell Giada to skip the manifest and just try detecting the plugin features by parsing the DLL?

Thanks

**Comment by gvnnz** [comment, marked as answer]
Hey @anymaw, VST2 are not supported anymore. You can change that by compiling Giada yourself with the right headers from the VST2.x SDK. More info here: https://giadamusic.com/documentation-compiling-from-source

**Reply by ghost** [reply]
I see. Well, fuck steinberg, and thank you!
